tests/test_database_to_staging.py

```diff
@@ -106,3 +109,735 @@ def test_store_assay_info_files(syn):
             used=f"{FILEVIEW_SYNID}.2",
         )
         assert wes_ids == ["A"]
+
+
+def get_run_genie_filters_test_cases():
+    return [
+        {
+            "name": "staging_true",
+            "genie_version": "STAGING",
+            "variant_filtering_synId": "synZZZZZ",
+            "clinicaldf": pd.DataFrame(dict(SAMPLE_ID=["SAGE-1"])),
+            "beddf": pd.DataFrame(dict(SAMPLE_ID=["TEST-1"])),
+            "center_mappingdf": pd.DataFrame(dict(vcf2maf=["synZZZZZ"])),
+            "processing_date": "TEST_DATE",
+            "skip_mutationsincis": True,
+            "consortium_release_cutoff": "CUTOFF_DATE",
+            "test": False,
+            "current_release_staging": True,
+        },
+        {
+            "name": "testing_true",
+            "genie_version": "TEST",
+            "variant_filtering_synId": "synZZZZZ",
+            "clinicaldf": pd.DataFrame(dict(SAMPLE_ID=["SAGE-1"])),
+            "beddf": pd.DataFrame(dict(SAMPLE_ID=["TEST-1"])),
+            "center_mappingdf": pd.DataFrame(dict(vcf2maf=["synZZZZZ"])),
+            "processing_date": "TEST_DATE",
+            "skip_mutationsincis": True,
+            "consortium_release_cutoff": "CUTOFF_DATE",
+            "test": False,
```

Comment:
Good catch, this test case should have testing : True to test the condition that MAFinBED.R will use the **test** `databaseSynIdMappingId`. 

The test case above this would test that MAFinBED.R will use the **staging** `databaseSynIdMappingId`.